app/assets/javascripts/discourse/app/components/post/menu.gjs

```diff
@@ -448,17 +449,27 @@ export default class PostMenu extends Component {
     );
   }
 
+  @cached
+  get checkLikedPost() {
+    if (this.currentUser && this.keyValueStore) {
```

Comment:
As suggested by david, this shouldn't be in a getter (it doesn't return anything! 😉) - constructor would be a better place.

```suggestion
    if (this.currentUser) {
```

`keyValueStore` service is guaranteed to be present.